STEERING-COMMITTEE.md

```diff
@@ -98,30 +108,11 @@ first name):
 | <img width="30px" src="https://github.com/evankanderson.png"> | Evan Anderson           | VMware       | [@evankanderson](https://github.com/evankanderson) | 2023-11-28 | 2025     |
```

Comment:
unrelated, but @evankanderson do you want to update your company affiliation too?

Reply:
Oops, thank you!

Reply:
In fact, I wasn't working at VMware when I was elected...